torch/onnx/utils.py

```diff
@@ -351,7 +355,7 @@ def _export(model, args, f, export_params=True, verbose=False, training=False,
     return torch_out
 
 
-def _set_input_and_output_names(graph, input_names, output_names):
+def _set_input_and_output_names(graph, input_names, output_names, params_dict):
```

Comment:
We don't need to change this function anymore.

Reply:
Reverted.